careers/jobs/senior-fullstack-node-engineer.md

```diff
@@ -0,0 +1,101 @@
+
+- Lead and advise on entire projects
+- Be hands-on in new and existing code bases
+- Working with the existing team to set and maintain standards of code quality and best practice across the organization
+- Work closely with the product team to understand, scope and advise on the best approach to achieve a business objective
+- Work with client’s engineering teams; leading, advising, and contributing to their solutions
+
+## Day to Day Tasks
+
+- Discuss and design new systems from the ground-up with strong opinions about software and design principles
+- Develop new and maintain existing systems
+- Write documentation to record and explain decisions around architecture and design
+- Write unit integration tests as a first-class citizen
+- Perform code reviews and participate actively in discussions, even around technologies you may not be an expert in
+
+## About You
+
+Please see our [Developer Proficiency](../../handbook/professional-development/developer-proficiency.md) for a full list of what we think makes a "Senior" developer, but some key call-outs are:
+
+- Experienced and comfortable through the entire life-cycle of a feature, from ideation to delivery
+- Can understand business drivers and make solid proposals for building new features or refining existing ones
+- Are a subject matter expert in at least one programming environment
+- Have typically 5-8 years experience in a specific programming domain or language
+- Excellent communications skills (native or strong English speaker)
+- Experienced working directly with clients or key stakeholders
+- Are ambitious and excited to grow with DVELP
+- Must be willing to travel to client locations when required
+
+### Technical Skills
+
+While we understand that good programmers can learn new skills, this role is ideally for someone who has good working knowledge and experience in the following:
+
+- React
+- Redux
+- Typescript
+- Node (Express)
+- GraphQL
+
+More generally, desirable but not required are:
+
+- Experience with Twilio APIs
+- Experience with Google AI APIs
+
+We know there's a lot more to programming than what's listed above - but we'd rather learn through talking where your strengths are, than prescribe a pre-set list of hard requirements.
+
+## Salary
+
+We've chosen not to disclose the sort of salary we're looking to pay, because everyone is individual and we want to give everyone what we think is very fair compensation. We want to have that conversation with you to ensure everyone's happy, rather than sticking to a number on a page.
+
+## Benefits
+
+There are a wide array of perks available to you as a DVELP employee - everything from cheaper cinema tickets to discounted getaways, and loads in between. We're also open to having conversations around what you may need that's specific to you - don't be afraid to ask.
+
+## Pension
+
+Upon finishing probation you will be automatically enrolled in the government's Workplace Pension scheme.
```

Comment:
Probably need to adjust this to align with sabio scheme

Reply:
Gotcha. Do you have details about this or will Ruby have this info?

Reply:
*Update:* Awaiting details from George